react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import reactDOM from 'react-dom/client';
+const container = document.querySelector('#root');
+const root = reactDOM.createRoot(container);
+
+const colors = {
+  0: 'dimgrey',
+  3: 'navy',
+  6: 'darkmagenta',
+  9: 'indianred',
+  12: 'lightsalmon',
+  15: 'palegoldenrod',
+  18: 'mintcream',
+
+};
+
+function CustomButton() {
+  const [count, setCount] = useState(0);
+  const handleClick = () => {
+    setCount(count + 1);
+  };
+
+  useEffect(() => {
```

Comment:
We'll study `useEffect` later. You shouldn't need to use `useEffect` here. There is a way to get a reference to the document element (`useRef`), but that's not needed, either. Instead, you use a css class for each color and dynamically set the class name.